test/ibmq/test_basic_server_paths.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Tests that hit all the basic server endpoints using both a public and premium provider."""
+
+import time
+
+from qiskit.test import slow_test
+from qiskit.providers.jobstatus import JOB_FINAL_STATES, JobStatus
+
+from qiskit.providers.ibmq import least_busy
+from ..decorators import requires_providers
+from ..ibmqtestcase import IBMQTestCase
+from ..utils import cancel_job, bell_in_qobj
+
+
+class TestBasicServerPaths(IBMQTestCase):
+    """Test the basic server endpoints using both a public and premium provider."""
+
+    @classmethod
+    @requires_providers
+    def setUpClass(cls, providers):
+        # pylint: disable=arguments-differ
+        super().setUpClass()
+        cls.providers = providers  # Dict[str, AccountProvider]
+
+    @slow_test
+    def test_job_submission(self):
+        """Test running a job against a device."""
+        for _desc, provider in self.providers.items():
+            backend = least_busy(provider.backends(
+                simulator=False, filters=lambda b: b.configuration().n_qubits >= 5))
+            provider_backend = {'provider': provider, 'backend': backend}
+            with self.subTest(provider_backend=provider_backend):
```

Comment:
Probably not that big a deal, but I think this will expose the premium h/g/p in the Travis log? Why not use `_desc` instead of `provider` in `provider_backend`?

Reply:
Yeah, that's a good point! I'll go ahead and change that.